src/utils/convertModifiedColorToHex.ts

```diff
@@ -1,29 +1,23 @@
-import chroma from 'chroma-js';
-import Color from 'colorjs.io';
+import Color from 'colorjs.io/dist/color.legacy';
```

Comment:
The babel loader should fix our issues for us
```suggestion
import Color from 'colorjs.io';
```